.circleci/config.yml

```diff
@@ -611,10 +578,6 @@ workflows:
           requires:
             - build-push-3scale-images
       - install-operator
```

Comment:
why leave install-operator job?

Reply:
I preferred to leave it as it can be executed properly, even if redundant due to done in e2e tests. It is also used in the "operator-release" and "nightly" jobs

Reply:
I am not saying deleting the job itself. I am suggesting removing from the nightly and operator workflows as it is useless.

Reply:
For nightly I think it might be useful. Same for operator-release job.
It's better than nothing.

For the "operator" job we could remove it as it's already covered by the OpenShift CI tests but anyway it did not harm and runs quick

Reply:
If you mean building the operator image as part of the nightly workflow, fine. But install-operator job does more than that and it it useless. Check it, exporting a whole docker image to tarball and persist it to workspace for nothing.

Reply:
In the "operator-release" workflow those additional steps you mentioned are actually needed.

In the nightly job those additional steps are not needed. I'll change the call to install-operator to build-operator-image and I'll remove it from the operator job as it's already covered by OpenShift CI tests

Reply:
I've applied the changes I've described

Reply:
yes, I never said to delete the install-operator job or remove it from the `operator-release` workflow

Reply:
:+1: 

Take a look at the current changes